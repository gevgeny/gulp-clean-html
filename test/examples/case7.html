<div>
    <div class="to-remove">
        node1
    </div>
    <div class="not-to-remove">
        node2
    </div>
    <div class="to-remove">
        node3
    </div>
    <div class="not-to-remove">
        node4
    </div>
</div>