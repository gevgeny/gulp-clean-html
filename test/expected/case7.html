<div>
    
    <div class="not-to-remove">
        node2
    </div>
    
    <div class="not-to-remove">
        node4
    </div>
</div>